index.html

```diff
@@ -74,6 +75,220 @@ <h1 class="main-text">I’m Giovanni Glad to see you!</h1>
             </div>
         </section>
     </div>
-   
+
+    <!-- Work Section -->
+    <section>
+        <div class="work-container">
+            <div class="work-container-background">
+                <div class="work-image-container">
+                    <img src="./Assets/mobile_Snapshoot_1.png" alt="">
+                </div>
+    
+                <div class="work-text-container">
+                    <h1>Tonic</h1>
+                    <div class="work-complementary-text">
```

Comment:
- [ ] Kindly use `h2` instead of `h1` to follow the html heading hierarchy 😊 due to SEO purpose 👍 Also there should be only one `h1` tag in a webpage 👍